includes/utils/class-amp-validation-utils.php

```diff
@@ -590,6 +590,7 @@ public static function print_edit_form_validation_status( $post ) {
 
 		// If no results from URL are available, validate post content outside frontend context.
 		if ( empty( $validation_errors ) && post_type_supports( $post->post_type, 'editor' ) ) {
+			do_action( 'pre_amp_render_post', $post->ID );
```

Comment:
Actually this changes can be reverted because this entire `if` statement is going to be removed in https://github.com/Automattic/amp-wp/pull/1132

Reply:
Just saw that now, we can drop my last 2 commits, sorry for the noise.